packages/cardano-services/src/StakePool/HttpStakePoolMetadata/HttpStakePoolMetadataService.ts

```diff
@@ -62,12 +62,7 @@ export const createHttpStakePoolMetadataService = (
       throw error;
     }
   },
-
   async getStakePoolMetadata(hash, url) {
```

Comment:
Note for future PRs: this refactor should probably be in a separate commit. Makes it easier to review and reason about changes by looking at commit log.

Reply:
https://github.com/input-output-hk/cardano-js-sdk/pull/923/files/ed8aafbe54b047c07ce0c4c0a39f66c90e6467a7#r1332944865 :grin: